File: lab1.1/src/Main2.java
import java.util.Scanner;

public class Main2 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n;

        System.out.print("Enter an integer greater than 999: ");

        if (scanner.hasNextInt()) {
            n = scanner.nextInt();

            if (n > 999) {
                int digit = (n / 100) % 10;
                System.out.println("The hundreds digit is: " + digit);
            } else {
                System.out.println("Error: Number must be greater than 999.");
            }
        } else {
            System.out.println("Error: You must enter an integer.");
        }

        scanner.close();
    }
}

File: lab1.1/src/Main2.kt
fun main() {
    print("Enter an integer greater than 999: ")
    val input = readLine()

    val n = input?.toIntOrNull()

    if (n == null) {
        println("Error: You must enter an integer.")
    } else if (n <= 999) {
        println("Error: Number must be greater than 999.")
    } else {
        val digit = (n / 100) % 10
        println("The hundreds digit is: $digit")
    }
}